sdk/apis/tenancy/v1alpha1/types_workspace.go

```diff
@@ -174,6 +176,23 @@ type WorkspaceSpec struct {
 	//
 	// +kubebuilder:format:uri
 	URL string `json:"URL,omitempty"`
+
+	// Mount is a reference to a mount that is used to mount the workspace.
+	// If specified, logicalcluster will not be created and the workspace will be mounted
+	// using reference mount object.
+	//
+	// +optional
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="mount is immutable"
```

Comment:
is this enough to stop you from removing the field? I think we need something like line 153 (on the higher level) with `has`.

Reply:
Added. Somehow, it fails to start when I add a filter to `namespace` Do you have any idea why the `omit empty` optional field fails to start?